internal/encoding/errors.go

```diff
@@ -2,5 +2,56 @@ package encoding
 
 import "errors"
 
+const (
+	//Executor codes
+
+	CodeOk                          = 0
+	CodeImproperTxnQueryRequest     = 6
+	CodeTxnQueryError               = 7
+	CodeImproperTxnHistoryRequest   = 8
+	CodeTxnRange                    = 9
+	CodeTxnHistory                  = 10
+	CodeImproperTypeURLRequest      = 11
+	CodeInvalidURL                  = 12
+	CodeTypeURL                     = 13
+	CodeImproperDirectoryURLRequest = 14
+	CodeDirectoryURL                = 15
+	CodeImproperChainIdRequest      = 16
+	CodeChainIdError                = 17
+	CodeDefaultError                = 18
+	CodeRoutingChainId              = 19
+	CodeCheckTxError                = 20
+	CodeUnsupportedTxType           = 21
+	CodeImproperTxnRequest          = 22
+	CodeTxnCheckFailed              = 23
+	CodeDeliverTxError              = 24
+	CodeImproperAcceptedTxn         = 25
+	CodeImproperPendingTxn          = 26
+	CodeTxnStateError               = 27
+	CodeRecordTxnError              = 28
+	CodeSyntheticTxnError           = 29
+	CodeMarshallingError            = 30
```

Comment:
Add a comment to each error code describing under what conditions it is returned (using the standard Go comment pattern). For example, "CodeUnsupportedTxType is returned when the transaction type is not supported".